### ISSUE TITLE
Ambiguous whether it is for Nuxt3 or not + Please lock versions for `package.json`

### ISSUE THREAD

**Issue by nopeless** [nuxt-community/express-template#211]
The package.json currently states that it uses nuxt@latest, yet there is no CI that checks whether it works with the latest build

I would like to discuss solutions to this

**Comment by kami16ru** [comment]
You are completely right! I changed in package.json nuxt 'latest' to '2' and project started. As you can see branch master last change was 2 years ago and if it works with nuxt 3, it works only on specific version with other packages 'latest' tag
